org.openhab.binding.kostalinverter/src/main/resources/ESH-INF/thing/PIKO1020.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="kostalinverter"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+    <thing-type id="kostalinverterpiko1020">
```

Comment:
The binding definition xml files should use tabs (not spaces) for indentation. Frankly this should have showed up as an error when you tried to build this project with maven so I'm a little surprised it hasn't been fixed already...